lib/auth/user_agent.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Auth.UserAgent do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Auth.Repo
+
+  schema "user_agents" do
+    field :user_agent, :string
+    has_many :login_logs, Auth.LoginLog
+  end
+
+  @doc false
+  def changeset(user_agent, attrs) do
```

Comment:
having the name of the schema be the same as the primary field is quite confusing.